messaging/broker.go

```diff
@@ -13,23 +12,34 @@ import (
 	"sort"
 	"strconv"
 	"sync"
+	"time"
 
+	"github.com/boltdb/bolt"
 	"github.com/influxdb/influxdb/raft"
 )
 
-// BroadcastTopicID is the topic used to communicate with all replicas.
-const BroadcastTopicID = uint64(0)
+// DefaultPollInterval is the default amount of time a topic reader will wait
+// between checks for new segments or new data on an existing segment. This
+// only occurs when the reader is at the end of all the data.
+const DefaultPollInterval = 100 * time.Millisecond
```

Comment:
This is a stop gap solution. A better solution is to only allow streaming topic readers to be created from topics and then have the topics notify the readers when there's an update to a topic through a channel. That would get rid of a ton of overhead in reading the directory, creating segments, matching segments, etc.